test/exchangeIssuance/exchangeIssuance.spec.ts

```diff
@@ -499,7 +498,9 @@ describe("ExchangeIssuance", async () => {
       context("when input erc20 token is weth", async () => {
         cacheBeforeEach(async () => {
           subjectInputToken = weth;
-          await subjectInputToken.connect(subjectCaller.wallet).approve(exchangeIssuance.address, MAX_UINT_256, { gasPrice: 0 });
+          await subjectInputToken
```

Comment:
FYI: I had to change all of these tests because I updated hardhat (to be able to fork the latest mainnet block, see integration tests below), which meant that we couldn't use "gasPrice: 0" anymore. So wherever we set `gasPrice: 0` in the tests to check ether balance changes accurately, I instead used my newly defined helper function to get the transaction fee and account for that in the tests.